### ISSUE TITLE
The Unseen Factors of Unreplicable Success

### ISSUE THREAD

**Issue by venezuela01** [loki-project/loki-improvement-protocols#39]
Allow me to share four business stories. These stories are based on real world cases but the details have been intentionally blurred or altered to maintain confidentiality. In fact, I personally know one of the founders from these stories.

1. Alice established a specialized tool site, for instance, a PDF digital form and signature tool, sold at a competitive rate for a pro version. Bob, thinking it would be straightforward to replicate, used his technical skills to build a similar site. However, despite its equal functionality, he had difficulty in generating sales, leaving him mystified.

2. Alice made profitable sales of a product on an online platform like Amazon. Bob, leveraging his close ties with manufacturers, listed the identical product on the same platform. Nonetheless, he was unable to match Alice's sales and profit margins, leaving him perplexed.

3. Alice initiated a successful crowdfunding campaign on Kickstarter, amassing $1.2 million from over 10,000 backers, thus marking her project as one of the platform's most triumphant. Bob, spurred by Alice's success, also launched a project on Kickstarter but received minimal support, leaving him dumbfounded.

4. A doggy cryptocurrency or NFT project reached remarkably high market capitalization and trade volume, leaving another project, which offered genuine value to users, perplexed due to their comparatively low market capitalization.

Let’s pause for a moment and try to decipher what transpired in these four cases.

Pause.

Now, please allow me to offer an explanation based on real-world examples:

1. In the first situation, Alice's success hinged on her substantial investment in marketing, like Google Ads. In fact, her marketing spent might account for 50% or even 80% of total income. As a result, a constant inflow of new users served as a barrier for competitors like Bob, an unforeseen "hidden cost" for newcomers. Bob's failure to identify this tactic led to his early surrender, leaving Alice to uphold her industry dominance.

2. Similarly, the secret behind Alice's success in the Amazon case was her hefty marketing investment within Amazon's internal channels. This drove considerable traffic to her listing, creating an illusion of tremendous profitability. Bob didn't perceive Alice's hidden marketing cost, leading to his inability to match her sales volumes and profitability.

3. Once again, Alice's success in the Kickstarter scenario resulted from her decision to hire a professional marketing team to promote her project. She likely invested $1 million in marketing before raising $1.2 million through crowdfunding. This was an unanticipated cost that Bob overlooked in his own approach. Despite this substantial outlay, Alice still managed to realize a 20% return on investment, making it a wise investment. Alice subsequently leveraged her crowdfunding success to appeal to venture capitalists and managed to raise an additional $10 million.

4. The mysterious cryptocurrency or NFT project heavily relied on aggressive marketing to inflate its market cap and trading volume, bearing a hidden cost. They likely operated at a loss for most of the year, only recuperating their losses during bullish runs by selling off their holdings.

In each case, the "hidden cost" served as the unseen force propelling success. Ignorance of these hidden costs can lead to miscalculations and failures, as exemplified by Bob's experiences.

When designing strategies like [ORC 8](https://github.com/oxen-io/oxen-improvement-proposals/issues/36), it's crucial to predict the return on investment (ROI) before implementation. It would be naive to ignore any potential hidden costs and assume that funding could be easily raised. Conversely, if a third-party marketing agent promises they can help raise our market cap, we need to carefully evaluate any potential hidden costs and the real ROI. Otherwise, we might end up losing money. In such scenarios, the third-party marketing agent might be the only one profiting. 

The idea of a 'minimal profitable product' is important to Oxen’s success. If the ROI is positive at a small scale, then by increasing marketing expenditure, we can scale up the profits until the market is saturated. However, if the ROI is negative from the start, we need to be very cautious about increasing marketing spend, as we might end up losing more money. Instead, we might need to explore and test different approaches to make money until we build a 'minimal profitable product'.

**Comment by KeeJef** [comment]
Yes i agree with these ideas @venezuela01, marketing will be very important to the long term success of the project, and we aren't discounting the costs associated with this. We are working to ensure that ORC-8 is well received by the market, and making any tweaks before embarking on the journey of promoting ORC-8 is essential. Thus far most external partners we have engaged with have had a positive view of ORC-8 and see it as reducing barriers to entry, unifying branding and allowing Session to become more interoperable. 

We are quite lucky to have a product like Session which is already growing without any direct advertising costs and has some smaller monetization features like ONS which we can show off. I think this has been beneficial, especially when compared to other projects which are earlier stage or require large marketing spends to experience growth.
